website/package.json

```
@@ -9,9 +9,10 @@
     "build": "node ace build",
     "dev": "node ace serve --hmr",
     "test": "node ace test",
-    "lint": "eslint .",
+    "lint": "eslint . --fix",
```

I don't like linting rules changing my code without me being aware of it, especially because, when I run `pnpm run lint`, this will run in the entire repository.

```suggestion
    "lint": "eslint .",
    "lint:fix": "eslint . --fix"
```